src/main/java/net/robinfriedli/aiode/Aiode.java

```diff
@@ -208,7 +208,7 @@ public static void shutdown(int millisToWait, @Nullable List<CompletableFuture<M
             try {
                 Thread.sleep(millisToWait);
             } catch (InterruptedException e) {
-                return;
+                Thread.currentThread().interrupt();
```

Comment:
Do you need to include the `return;` statement here?

Reply:
Yes. Good catch!